### DISCUSSION TITLE
Verifying teacher email

### DISCUSSION THREAD

**Discussion by MarkStandley** [community/community#121173] [GitHub Education]
### Hi everyone,

Hi
Our students love using github and verify themselves on the domain @student.collegiate.school.nz (which seems to be linked to our school).  However, teachers - like me - are on the domain @collegiate.school.nz.  How do I verify myself when I don't have an email with the "student" bit.
Thanks for any help anyone can give me in advance.

**Comment by samus-aran** [comment]
Hi @MarkStandley  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
